CUJExecutor/megaTest

```diff
@@ -1,7 +1,25 @@
 #!/usr/bin/env bash
 
-##Loops through all provided compilation flags 10 times and runs manipulateProfilesAndTest
-##Appends the SHORT_SUMMARY from each run to the AGGREGATED_DATA_FILE for the specified INPUT_FILE
+##
+# Loops through all provided compilation flags 5 times, each time recompiling aGMM as specified and measuring the desired portion of the CUJ specified by INPUT_FILE
```

Comment:
Should end the proper sentence with a period (judging from other sentences that are ended with periods).